__tests__/external/shared/serializers/json-api-serializer/associations/includes-test.js

```diff
@@ -519,4 +519,70 @@ describe("External | Shared | Serializers | JSON API Serializer | Associations |
       );
     }).toThrow();
   });
+
+  test(" Do not return includes when queryParamIncludes is empty ", () => {
+    server.config({
+      serializers: {
+        application: JSONAPISerializer
+      }
+    });
+
+    let post = server.schema.blogPosts.create();
+
+    let request = {
+      queryParams: {
+        include: " "
+      }
+    };
+
+    let result = server.serializerOrRegistry.serialize(post, request);
+
+    expect(result).toEqual({
+      data: {
+        type: "blog-posts",
+        id: "1",
+        attributes: {}
+      }
+    });
+  });
+
+  test("Do not throw when queryParamIncludes has a trailing comma", () => {
```

Comment:
Could we move these tests to the higher level route handler test below? I don't want us to miss something that we might not be catching it here since we're calling `serialize()` method directly instead of sending a request like `/posts/1?include=comments,` to the server.

I think two tests for this PR would suffice, one you basically have already down there. Just can move the trailing comma one there too. And I think add assertion against the data returned.